web/src/client/storage.js

```diff
@@ -645,8 +650,8 @@ class ProposalManager {
     };
 
     return {
-      target: dbusVolume.Target.v,
-      targetDevice: dbusVolume.TargetDevice?.v,
+      target: dbusVolume.Target?.v || "default",
```

Comment:
NP: I guess we can assume this value always comes from D-Bus.

Reply:
Fine with me